Disable hidden attribute to prevent linker error:
ld: fatal: relocation error: R_386_GOTOFF: file indirect_init.o: symbol __indirect_glNewList: relocation must bind locally


--- src/glx/x11/indirect.h	2006-08-25 15:09:00.000000000 -0700
+++ src/glx/x11/indirect.h	2006-11-08 16:18:20.013840000 -0800
@@ -37,7 +37,7 @@
  * \author Ian Romanick <idr@us.ibm.com>
  */
 
-#  if (__GNUC__ > 3 || (__GNUC__ == 3 && __GNUC_MINOR__ >= 3)) && defined(__ELF__)
+#  if 0
 #    define HIDDEN  __attribute__((visibility("hidden")))
 #  else
 #    define HIDDEN
